### PR TITLE
Add VIA Functionality for Keyhive Maypad

### PR DESCRIPTION
**Pull request by ilikepi2**
## Description

Added VIA Functionality for Keyhive Maypad

Added VIA functionality for the Maypad. This keymap is for the Ortho_5x4 layout.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keyhive/maypad/config.h [hunk 1]
```diff
@@ -17,7 +17,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4B48 //KH for Keyhive
 #define PRODUCT_ID      0x6060
```
Comment (fauxpark) [inline keyboards/keyhive/maypad/config.h:21]:
```suggestion
#define PRODUCT_ID      0x4D50 // MP
```

File: keyboards/keyhive/maypad/keymaps/via/config.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2019 codybender
```
Comment (fauxpark) [inline keyboards/keyhive/maypad/keymaps/via/config.h:1]: This file can be deleted.

File: keyboards/keyhive/maypad/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+}
+
+void matrix_scan_user(void) {
+
+}
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
Comment (fauxpark) [inline keyboards/keyhive/maypad/keymaps/via/keymap.c:41]:
```suggestion
    LAYOUT_ortho_5x4(
        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
        KC_P4,   KC_P5,   KC_P6,   KC_PPLS,
        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
        KC_P0,   KC_P0,   KC_PDOT, KC_PENT
    )
};
```
Generally the default VIA keymap should look as close as possible to the default keymap.

File: keyboards/keyhive/maypad/keymaps/via/rules.mk [hunk 4]
```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LT0_ENABLE = yes
+RAW_ENABLE = yes
+DYNAMIC_KEYMAP_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/keyhive/maypad/keymaps/via/rules.mk:4]:
```suggestion
LTO_ENABLE = yes
```

File: keyboards/keyhive/maypad/keymaps/via/config.h [hunk 5]
```diff
@@ -0,0 +1,19 @@
+/*
```
Comment (fauxpark) [inline keyboards/keyhive/maypad/keymaps/via/config.h:1]: I mean the entire file, not just this line.

Comment (ilikepi2) [reply keyboards/keyhive/maypad/keymaps/via/config.h]: Gotcha, I've removed the config.h file from keymaps/via

**Comment by ilikepi2** [comment]
Thank you so much for all of your help so far, @fauxpark.

This is my first attempt at a contribution, I appreciate all the help I can get!

**Comment by Erovia** [comment]
Thank you for your contribution.

**Pull request merged**